### DISCUSSION TITLE
Allow selection of multiple archives

### DISCUSSION THREAD

**Discussion by XXXBold** [borgbase/vorta#846] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not a problem, but would improve usability. On the "Archive" Tab, there's currently only one archive selectable at once. it would be much more efficient for e.g. deleting when one could select multiple entries.

**Describe the solution you'd like**
Allow multiple selections for the archive list. Maybe disable (grey out) the "Extract" and "Mount" options if more than one entry is selected, because that wouldn't make sense to run on multiple entries?

**Describe alternatives you've considered**
Select one by one and do the requested operation, which is very annoying if a lot of entries are present.

**Comment by ghost** [comment]
Yes please! Deleting old archives one-by-one is extremely tedious.

**Reply by m3nu** [reply]
This is already implemented.

![Screenshot 2024-02-19 at 07 29 34](https://github.com/borgbase/vorta/assets/3916435/09e4cd11-7713-4b91-9366-9d95b38a331f)

**Reply by ghost** [reply]
Argh, sorry. I'm using an old version, didn't realise.
